### DISCUSSION TITLE
Can you identify the sulci by looking at a photo of part of the surface of a 3D cerebral model?

### DISCUSSION THREAD

**Discussion by YoshiTaku17** [brainvisa/brainvisa.github.io#170] [Q&A]
Can you identify the sulci by looking at a photo of part of the surface of a 3D cerebral model?

I would like to compare the 3D models of the cerebrum created in the surgical simulation. The method of comparison is the sulci of the opercular area. I would like to compare which of the 3D models ① and 3D models ② more clearly depicts the sulci in the opercular area.

**Comment by denisri** [comment]
Hi,
I'm not sure to totally understand the question. Actually we cannot see your images models 1 and 2, so I may misunderstand the question... You are asking if it is possible (with a neuroanatomical point of view) to identify sulci when just seeing a part of the brain surface, is it that ?
We are not used to look at it this way here so I would not give a definitive answer, and I guess it depends on the brain regions, some are "easier" than others, but in a general way I think you need context to identify a sulcus, and the less you see, the more it is error-prone. Even when looking at the full brain we are not always sure, so when only looking at a small part, identifications are likely to be wrong in many cases...

Denis
